### ISSUE TITLE
New delete job running before delta finishes 

### ISSUE THREAD

**Issue by aviramradai** [agibralter/ts-resque-delta#27]
Hi pat,

Thanks for tall the great work you are doing.
I tried to upgrade to the new version of ts-resque-delta, when I updated a record I got 2 jobs under the ts_delta queue. One for building the delta and one for deleting from the core index. What will happen if the delete will finish running before the delta? Will I lose records while searching while the delta job is running?

Thanks,
Aviram

**Comment by pat** [comment]
That is correct, values will not appear if they're marked as deleted in the core index but not yet in the delta index. Hopefully this would not have an impact provided the delta updates are quick (which is aided by running a full index regularly as well).
